packages/material-ui/src/Select/SelectInput.js

```diff
@@ -139,21 +130,6 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     }
   };
 
-  const handleBlur = event => {
-    if (ignoreNextBlur.current === true) {
-      // The parent components are relying on the bubbling of the event.
-      event.stopPropagation();
-      ignoreNextBlur.current = false;
-      return;
-    }
-
-    if (onBlur) {
-      event.persist();
-      event.target = { value, name };
```

Comment:
This part might be important for formik and react final form integration. I'm not 100% sure either. It was introduced in #12204.

Reply:
Yeah that seems like the only part actually required.

Reply:
Restored